soci/soci_index.go

```diff
@@ -346,8 +368,8 @@ func (b *IndexBuilder) buildSociLayer(ctx context.Context, desc ocispec.Descript
 		return nil, errNotLayerType
 	}
 	// check if we need to skip building the zTOC
-	if skipBuildingZtoc(desc, b.config) {
-		fmt.Printf("layer %s -> ztoc skipped\n", desc.Digest)
+	if skip, reason := skipBuildingZtoc(desc, b.config); skip {
+		fmt.Printf("ztoc skipped - layer %s (%s) %s\n", desc.Digest, desc.MediaType, reason)
```

Comment:
I think we might want to keep the format:
```
fmt.Printf("layer %s -> ztoc skipped: %s\n",  desc.Digest, reson)
```

Reply:
The `ztoc skipped - ` format is taken from your suggestion in #591.

Reply:
Agreed with conforming the feature request.